src/lib/storage/table.cpp

```diff
@@ -22,6 +22,34 @@
 #include "utils/assert.hpp"
 #include "value_segment.hpp"
 
+namespace {
+
+using namespace hyrise;  // NOLINT(build/namespaces)
+
+// Checks if the two vectors have common elements, e.g., lhs = [0, 2] and rhs = [2, 1, 3].
```

Comment:
What are expected cardinalities here?

Reply:
Very few. In most cases just 1. Also, we don't add constraints over and over but maybe 2 or 3 ODs per selected dimension table, one or two UCCs, and one IND.

Reply:
Yeah, the overhead isn't worth if for such small values. Okay.